apps/api/src/app/workflows-v2/usecases/validate-content/prepare-and-validate-content/prepare-and-validate-content.usecase.ts

```diff
@@ -292,12 +297,48 @@ export class PrepareAndValidateContentUsecase {
         controlValueToContentIssues,
         item,
         StepContentIssueEnum.MISSING_VALUE,
-        `${toSentenceCase(item)} is missing`,
+        `${toSentenceCase(item)} is required`,
```

Comment:
Align the Dashboard validation errors with API, used for the in-app body validation error: `Body is required`.